src/lorarmesh.cpp

```diff
@@ -1,5 +1,22 @@
+#include <Layer1_LoRa.h>
+#include <LoRaLayer2.h>
 #include "loramesh.h"
 
+#define ADDR_LENGTH 4
+#define LORA_CS 18
+#define LORA_RST 14
+#define LORA_IRQ 26
+#define LORA_FREQ 915E6
+#define LED 25
```

Comment:
Not used. Please remove.

Reply:
This section `#define LED 25` is the only one that isn't being used.

Reply:
The others are used to initialize the `Layer1->setPins`